cmd/controlplane/docs/docs.go

```diff
@@ -177,10 +201,10 @@ var doc = `{
                 "created_at": {
                     "type": "string"
                 },
-                "deleted_at": {
+                "domain": {
                     "type": "string"
                 },
-                "domain": {
+                "hashed_grant_token": {
```

Comment:
should we return id instead?

Reply:
I will remove it instead to keep the API simple for now